generate-packaging-dir

```diff
@@ -11,20 +27,23 @@ ID="$1"
 VERSION_ID="$2"
 
 DST="debian-$ID-$VERSION_ID"
-rm -rf $DST
+rm -rf "$DST"
 mkdir -p "$DST"
 
-if ! git remote | grep upstream; then
-    git remote add upstream https://github.com/snapcore/snapd
+tree="$(tree1404)"
+if [ -z "$tree" ]; then
+    if ! git remote | grep upstream; then
+        git remote add upstream https://github.com/snapcore/snapd
```

Comment:
any particular reason to do this and a full fetch, instead of specifying all the bits in the fetch command?